doctr/models/recognition/postprocessor.py

```diff
@@ -52,21 +52,18 @@ def ctc_decoder(
 
         """
         batch_len = tf.cast(tf.shape(logits)[0], dtype=tf.int64)
-        sequence_len = tf.cast(tf.shape(logits)[1], dtype=tf.int64)
+        sequence_len = tf.cast(tf.shape(logits)[1], dtype=tf.int32)
 
         # computing prediction with ctc decoder
-        decoded_logits = K.ctc_decode(
-            tf.nn.softmax(logits), sequence_len * tf.ones(shape=(batch_len,), dtype=tf.int64), greedy=True
+        _prediction = tf.nn.ctc_greedy_decoder(
```

Comment:
if this backward compatible with both TF2.3 and TF2.4?

Reply:
Yes, I ran it on 2.3.1 on my computer